controllers/front/CmsController.php

```diff
@@ -156,6 +156,9 @@ public function getTemplateVarPage()
             $page['body_classes']['cms-id-'.$this->cms_category->id] = true;
         } else {
             $page['body_classes']['cms-id-'.$this->cms->id] = true;
+            if (!$this->cms->indexation) {
+                $page['meta']['robots'] = 'noindex';
```

Comment:
The meta tag is only present if the value is different from `index`, its default value set in the `FrontController`. Without the meta, the page will be indexed by search engines automatically, but with the `noindex` it won't be indexed.